OsmAnd/res/values/strings.xml

```diff
@@ -55,9 +55,9 @@
     <string name="shared_string_other">Other</string>
     <string name="vehicle_parameters_descr">Weight, height, speed</string>
     <string name="vehicle_parameters">Vehicle parameters</string>
-    <string name="voice_announces_info">Voice announces are played only during navigation.</string>
-    <string name="voice_announces_descr">Navigation instruction and announces</string>
-    <string name="voice_announces">Voice announces</string>
+    <string name="voice_announces_info">Voice announcements only happen during navigation.</string>
+    <string name="voice_announces_descr">Navigation instruction and announcements</string>
+    <string name="voice_announces">Voice announcements</string>
```

Comment:
"<string name="interrupt_music_descr">Voice prompts pause, not just subdue, music playback.</string>" is problematic and wrong. Will tackle the "prompts" in next PR. I think many of them can be omitted, and otherwise clarified.

Reply:
Ok, we could postpone this.

Reply:
@sonora As in postpone the additional actions, or remove this for now?